net.openhabittracker.app.yaml

```diff
@@ -0,0 +1,85 @@
+id: net.openhabittracker.app
+#runtime: org.freedesktop.Platform
+#runtime-version: '23.08'
+#sdk: org.freedesktop.Sdk
+runtime: org.gnome.Platform
+runtime-version: '46'
```

Comment:
why do you need the GNOME runtime? (also 47 is out if you really need the GNOME runtime)

Reply:
this wasn't answered.

Reply:
I tried running my app with org.freedesktop but it didn't work.

When I changed it to org.gnome it started working.

I assume it has something to do with https://github.com/tryphotino/photino.Blazor and more specifically with https://github.com/tryphotino/photino.NET and https://github.com/tryphotino/photino.Native

Photino is using webkit2gtk to display web content in desktop apps and Photino.Blazor is using that to run Blazor in a desktop app.

My app is also using native file open and file save dialogs.

I use Photino.Blazor only for Linux, so I don't know much about it - I use Microsoft Maui for Windows, macOS, iOS, Android.